src/check_entity_ACL.c

```diff
@@ -0,0 +1,61 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   check_entity1.c                                    :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: gwolf <gwolf@student.42vienna.com>         +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2023/11/10 18:17:46 by gwolf             #+#    #+#             */
+/*   Updated: 2023/11/20 09:44:47 by gwolf            ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "check.h"
+
+t_type	ft_check_ambient(char *line)
+{
+	ft_rm_space(&line);
+	if (!ft_isvalid_float(&line, 0.0, 1.0, false))
```

Comment:
As i wrote in PR description: i decided against it, since it would be harder to integrate. 
I would need to create a t_interval or pass a compound literal, which usage is still questionable re norm conformity

Reply:
You can use my t_interval. After merge

Reply:
I'll try